### PR TITLE
feat: social accounts in popups

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MF-4202

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/background/tasks/Cancellable/PopupSSR/cache.ts [hunk 1]
```diff
@@ -52,13 +53,14 @@ async function prepareData(): Promise<PopupSSR_Props> {
     const id = await getCurrentPersonaIdentifier()
     const personas = await queryOwnedPersonaInformation(false)
     const currentPersona = personas.find((x) => x.identifier === id) || personas.at(0)
-
+    const networks = await getSupportedSites({ isSocialNetwork: true })
```
Comment (guanbinrui) [inline packages/mask/background/tasks/Cancellable/PopupSSR/cache.ts]: Can we use `Promise.all()`?

File: packages/mask/src/extension/popups/components/SocialAccounts/index.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                <Box className={classes.accountItem} key={index}>
+                    <AccountAvatar
+                        avatar={account.avatar}
+                        network={account.identifier.network}
+                        isValid={account.is_valid}
+                    />
+                    <Typography className={classes.identity}>@{account.identity}</Typography>
+                </Box>
+            ))}
+            <Box />
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/components/SocialAccounts/index.tsx]: Didn't match an open `<Box>`?

Comment (nuanyang233) [reply packages/mask/src/extension/popups/components/SocialAccounts/index.tsx]: work in progress

File: packages/mask/src/extension/popups/pages/Personas/AccountDetail/index.tsx [hunk 3]
```diff
@@ -108,27 +170,52 @@ const AccountDetail = memo(() => {
...
+                walletProofs={walletProofs}
+                isClean={isClean}
+                toggleUnlisted={toggleUnlisted}
+                listingAddresses={listingAddresses}
+                loading={isInitialLoading}
+                onSubmit={handleSubmit}
+                submitting={submitting}
             />
-            <DisconnectDialog
+            {/* <DisconnectDialog
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Personas/AccountDetail/index.tsx]: Remove commented code.

**Review by guanbinrui — DISMISSED** [review]

**Pull request merged**